Rakefile

```diff
@@ -644,21 +644,29 @@ private
 
 ## Before you start...
 
-We will be installing multiple applications.  The installation steps will be provided for each application.
+PLEASE READ THIS WHOLE PAGE! You're going to be installing a lot of stuff on your computer in a way you probably never have before. It's perfectly safe, but it's going to look scary!
 
-You should be able to copy and paste the lines into Terminal -- except for a few that have obvious prompts in them, like "YOUR NAME", which you should replace accordingly.
+- THE WAY THIS WORKS: This script is going to check for different pieces of software on your computer. If you don't have those pieces of software -- or you do, but they need to be tweaked -- the script will stop and tell you what to do. When you're done, the script will restart. This will continue until everything is set up correctly.
 
-Many instructions start with dollar a sign (`$`).  The dollar sign is a convention used to indicate a bash (terminal) command.  You **should not** actually write the `$`.
+- You should be able to COPY AND PASTE the lines into Terminal.
 
-We recommend that you configure your system so that you can see both the instructions and Terminal at the same time.
+- Copy and paste ONE LINE AT A TIME.
 
-## Open Terminal "app"
+- If you see words in all-capitals (like "YOUR NAME"), they're a placeholder that you should replace with whatever's appropriate.
 
-If you haven't done so already, open Terminal so you can begin entering commands.
+- Many instructions start with a dollar sign (`$`). This just indicates a new line.  You **SHOULD NOT** actually write the `$`.
```

Comment:
It does not indicate "a new line".  I'm not even sure what that means.  It is the industry convention for specifying  terminal/bash command.  Why candy coat it?  I felt this was a good opportunity to learn that.  In addition, how about something like, "In practice, this means that each "$" indicates the an individual command to be copied and pasted."